include/rmm/device_buffer.hpp

```diff
@@ -393,28 +393,5 @@ class device_buffer {
                                     ///< allocate/deallocate device memory
 };
 
-/**--------------------------------------------------------------------------*
- * @brief Copies rmm::device_buffer to a preallocated host buffer.
- *
- * Copies device memory asynchronously on the specified stream
- *
- * @throws std::runtime_error if `hb` is `nullptr` or copy fails
- *
- * @param db `rmm::device_buffer` to copy to host
- * @param hb host allocated buffer to copy data to
- * @param stream CUDA stream on which the device to host copy will be performed
- *-------------------------------------------------------------------------**/
```

Comment:
@jrhemstad mentioned offline that the documentation should stay in the header. So fixing this in PR ( https://github.com/rapidsai/rmm/pull/232 ).